CODIGO/Protocol.bas

```diff
@@ -3486,7 +3472,8 @@ Private Sub HandleShowMessageBox()
 
     mensaje = Reader.ReadString8()
 
-    Select Case QueRender
+    Select Case g_game_state.state()
+    
 
         Case 0
```

Comment:
can we replacle the values of case 0 with the constant?

Reply:
Done